test/unit/RefRed/peak_finder_algorithms/peak_finder_derivation_test.py

```diff
@@ -1,145 +1,129 @@
-import sys
```

Comment:
It may be better to rename this test to `test_peak_finder_deriva....py` to be consist with other tests' names.

Reply:
I just did